paimon-flink/paimon-flink-cdc/src/main/java/org/apache/paimon/flink/sink/cdc/NewTableSchemaBuilder.java

```diff
@@ -59,10 +64,20 @@ public Optional<Schema> build(RichCdcMultiplexRecord record) {
                         record.primaryKeys(),
                         Collections.emptyMap(),
                         null);
+        List<String> specifiedPartitionKeys = new ArrayList<>();
+        if (partitionKeys != null && !partitionKeys.isEmpty()) {
+            specifiedPartitionKeys = partitionKeys;
```

Comment:
if `partitionKeyMultipleList` is not null, maybe it is better to use it.

Reply:
I will make adjustments and prioritize using partitionKeyMultipleList